### PR TITLE
Beginning glow layer serialization

### PR DESCRIPTION
**Pull request by julien-moreau**
Need to check if "excludedMeshes" and "includedMeshes" must be deleted from the new GlowLayer instance due to the SerializationHelper.Parse(...):
delete gl.excludedMeshes;
delete gl.includedMeshes;

### CODE REVIEW

File: src/Layer/babylon.glowLayer.ts [hunk 1]
```diff
@@ -115,7 +119,7 @@
          * @param scene The scene to use the layer in
          * @param options Sets of none mandatory options to use with the layer (see IGlowLayerOptions for more information)
          */
-        constructor(public name: string, scene: Scene, options?: Partial<IGlowLayerOptions>) {
+        constructor(name: string, scene: Scene, options?: Partial<IGlowLayerOptions>) {
```
Comment (deltakosh) [inline src/Layer/babylon.glowLayer.ts]: This is a breaking change
You must comment it directly in the constructor

File: src/Layer/babylon.effectLayer.ts [hunk 2]
```diff
@@ -651,5 +662,13 @@
             this.onAfterComposeObservable.clear();
             this.onSizeChangedObservable.clear();
         }
+
+        /**
+         * Serializes this effect layer
+         * @returns - serialized effect layer object.
```
Comment (deltakosh) [inline src/Layer/babylon.effectLayer.ts]: Please do not use - but "a" in this case

File: src/Layer/babylon.glowLayer.ts [hunk 3]
```diff
@@ -441,5 +445,70 @@
...
+                    }
+                }
+            }
+
+            return serializationObject;
+        }
+
+        /**
+         * Creates a Glow Layer from parsed glow layer data.
+         * @param parsedGlowLayer - Parsed glow layer data.
```
Comment (deltakosh) [inline src/Layer/babylon.glowLayer.ts]: Please do not use "-" but "defines" in this case

File: src/Layer/babylon.glowLayer.ts (hunk 1 above)
Comment (julien-moreau) [reply src/Layer/babylon.glowLayer.ts]: Don't understand the breaking change :(
As we have "public name: string" in BABYLON.EffectLayer

Comment (deltakosh) [reply src/Layer/babylon.glowLayer.ts]: Right I missed it :)

File: src/Layer/babylon.glowLayer.ts [hunk 4]
```diff
@@ -441,5 +445,70 @@
...
+                    }
+                }
+            }
+
+            return serializationObject;
+        }
+
+        /**
+         * Creates a Glow Layer from parsed glow layer data.
+         * @param parsedGlowLayer Defines glow layer data.
```
Comment (deltakosh) [inline src/Layer/babylon.glowLayer.ts]: no period at the end and no capital at the beginning (sorry :))

Comment (julien-moreau) [reply src/Layer/babylon.glowLayer.ts]: No problems, pushing right now :)

File: src/Layer/babylon.glowLayer.ts [hunk 5]
```diff
@@ -441,5 +445,78 @@
...
+                }
+            }
+
+            return serializationObject;
+        }
+
+        /**
+         * Creates a Glow Layer from parsed glow layer data
+         * @param parsedGlowLayer defines glow layer data
+         * @param scene bjs scene
```
Comment (deltakosh) [inline src/Layer/babylon.glowLayer.ts]: defines

Comment (julien-moreau) [reply src/Layer/babylon.glowLayer.ts]: Oh everywhere!

Comment (deltakosh) [reply src/Layer/babylon.glowLayer.ts]: yes because of the way it is displayed in the doc :)

Comment (julien-moreau) [reply src/Layer/babylon.glowLayer.ts]: That's what I finally guessed ^^

**Comment by deltakosh** [comment]
Thanks!

**Pull request merged**